lambdas/transcode/index.py

```diff
@@ -7,14 +7,24 @@
 from t4_lambda_shared.decorator import api, validate
 from t4_lambda_shared.utils import get_default_origins, make_json_response
 
+VIDEO_FORMATS = ['mp4', 'webm']
+AUDIO_FORMATS = ['mp3', 'ogg']
```

Comment:
Can we support FLAC? WAV? (Customer requests)
Same question for MPEG-TS and M3U8 (might just work?). Some .TS examples here https://open.quiltdata.com/b/streaming-orcasound-net/tree/rpi_orcasound_lab/hls/1541027406/

OK to do these new formats in a separate PR or postpone until after re:Invent.

Reply:
> Can we support FLAC? WAV? (Customer requests)

For output?

Reply:
No for input

Reply:
Oh, this is all about output formats. It'll support whatever ffmpeg supports as input - not sure if there's even a way to specify it.

Reply:
OK resolving this